README.md

```diff
@@ -78,6 +85,12 @@ To run `OpenNext` locally:
    path/to/open-next/packages/open-next/dist/index.js build
    ```
 
+### Coldstart
+
+OpenNext provide you with a warmer function that can be used to reduce cold start.
+
+On Lambda, there is multiple scenario where a lambda will trigger a cold start even if you have some warmed instance. For example if you have more request than warm instance you'll get a cold start. Also NextJs lazy load the routes, so even if you hit a warm instance, this specific route might not have been loaded yet.
```

Comment:
Thanks for the details!

Just a minor correction on the sentence: `there is multiple scenario` should be `there are multiple scenarios`